src/main/webapp/app/Utility/common.ts

```diff
@@ -0,0 +1,10 @@
+
+export class Common {
```

Comment:
Ok i have several questions to this part of the code 😆 

Why didn't you just export the function instead of a static class?
Why is the min date yesterday?
Why not 
`minDate.setDate(minDate.getDate() - 1);`
?

Additionally i think this method is to specific to be a static helper method. It should be implemented in the component. ( I am no Fan of static helper functions anyway)

Reply:
1 - I implemented  it as class because it has been used for any common features through out the  app if any, so in this perspective i used. (In this case two components are using this same method)
Yes you right it can be implemented through export function also. 
It's not yesterday , selection is allowed from today onward. It is library behaviour.